alfresco-search/src/main/java/org/alfresco/solr/SolrInformationServer.java

```diff
@@ -1674,6 +1657,88 @@ public void putAclTransactionState(UpdateRequestProcessor processor, SolrQueryRe
         }
     }
 
+    public void capIndex(long dbid) throws IOException
```

Comment:
Missing try finally for request and processor tidy up

Reply:
hmm, that's quite bad. I'll fix.